src/settingsdialog.ui

```diff
@@ -17,7 +17,7 @@
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
-      <number>0</number>
+      <number>4</number>
```

Comment:
Close the qt ui editor while having the general tab selected so that the default index doesn't change.